sessionctx/variable/tidb_vars.go

```diff
@@ -1258,6 +1260,7 @@ const (
 	DefTiDBSkewDistinctAgg                         = false
 	DefTiDB3StageDistinctAgg                       = true
 	DefTiDB3StageMultiDistinctAgg                  = false
+	DefTiDBOptExplainNoEvaledSubquery              = false
```

Comment:
Prefer `DefTiDBOptExplainEvaledSubquery = true`.
Although they mean the same thing. But the double negation of ```no xxx = false``` is more difficult to understand.